prow/gerrit/gerrit.go

```diff
@@ -294,14 +299,25 @@ func (c *Controller) ProcessChange(change gerrit.ChangeInfo) error {
 		if _, err := c.kc.CreateProwJob(pj); err != nil {
 			logger.WithError(err).Errorf("fail to create prowjob %v", pj)
 		} else {
-			triggered = append(triggered, spec.Name)
+			var b bytes.Buffer
+			if c.ca.Config().Plank.JobURLTemplate != nil {
+				if err := c.ca.Config().Plank.JobURLTemplate.Execute(&b, &pj); err != nil {
+					logger.WithFields(pjutil.ProwJobFields(&pj)).Errorf("error executing URL template: %v", err)
+				}
+			}
+			triggeredJobs = append(triggeredJobs, triggeredJob{Name: spec.Name, URL: b.String()})
 		}
 	}
 
-	if len(triggered) > 0 {
+	if len(triggeredJobs) > 0 {
 		// comment back to gerrit
+		message := "Triggered presubmit:"
+		for _, job := range triggeredJobs {
+			message += fmt.Sprintf("\n  * Name: %s, URL: %s", job.Name, job.URL)
```

Comment:
We should only print the URL part if `len(job.URL) > 0`.

Reply:
done